File: api/Oc6.Bold/Dtos/GameDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Linq.Expressions;
using System.Text;
using System.Threading.Tasks;

namespace Oc6.Bold.Dtos
{
    public record GameDto(int Id, string Name, bool IsActive, IEnumerable<TeamDto> Teams);
}

File: portal/src/models/GameDto.ts
import { TeamDto } from "./";

export interface GameDto {
    id: number;
    name: string;
    isActive: boolean;
    teams: Array<TeamDto>;
}

export const getDefaultGameDto: () => GameDto = () => ({
    id: -Math.random(),
    name: "",
    isActive: false,
    teams: []
});